pages/homepage/index.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+function Homepage() {
```

Comment:
We need the homepage to be served at endpoint of `/` and not `/homepage`, put this in `./pages/index.js` if you intended to make this component for actual homepage as it still have boilerplate nextjs stuff